values.yaml

```diff
@@ -89,19 +97,19 @@ tib:
     #  memory: 128Mi
     # requests:
     #  cpu: 100m
-    #  memory: 128Mi
-  nodeSelector: {}
-  tolerations: []
-  affinity: {}
-  extraEnvs: []
+  #  memory: 128Mi
+  nodeSelector: { }
+  tolerations: [ ]
+  affinity: { }
+  extraEnvs: [ ]
   configMap:
     #Create a configMap to store profiles json
     profiles: tyk-tib-profiles-conf
 
 dash:
   replicaCount: 1
   hostName: "localhost"
-  license: ""
+  license: "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhbGxvd2VkX25vZGVzIjoiOGU4NWFjZTUtNGRjOC00OTk1LTRkNzUtNWI4YTM2YzA4ZTJmLDViMDAzYzI1LTg3ZjYtNDIyZC00ODBhLWRiMGNjZDdhNjBlYyw2NWM4NDhhMy0xYWU4LTRhNzQtNGYwOS05ZjA5YmNkYjM5MGQsZmIwMTZhM2EtZjAxYi00ZWJiLTRlYzQtODI0NjkzYzE0ZWJmLDBkMDYwYTQ3LWUwMTktNDU1My01MGFhLWVjOWY4ZTEzZGE3YiwxNzdjMmZiZC04ZjRiLTQxZjEtNTg2ZC01MGE5YjQ0M2UzZWQsYjZjNGVjN2QtZGMwMi00NTg4LTcyMGMtY2FmNjRlNDEzYzE0LDNhMmJkYTBkLWY0M2ItNGY3OS00YzkzLWJkNDcxOTAzZDRiYSxlYWM5NDRlZC02NmVhLTRkNmUtN2IwYi05YmQ4N2VhZGRmNTQsZWJiNDk4MzQtOTFiNi00OTE5LTdkOTYtNWFkN2Y2ZjM0MTBiIiwiZXhwIjoxNjAxODMyOTMxLCJpYXQiOjE1OTkyNDA5MzEsIm93bmVyIjoiNTc3OTcxMTk0NWY5MmU2Njg5MDAwMTI3Iiwic2NvcGUiOiJtdWx0aV90ZWFtLHJiYWMsZ3JhcGgiLCJ2IjoiMiJ9.LnxOKxc_qVztRZ5cJM0Orog3-_eeco-caTy_QrbAvjdo_4G2r9Odk1KuGmLrW9fsjJZ-siALRb33DgPSQazEIaIgbVGbzp-r024Hc5mQuwZp2sZ9hBnxgra4xxvETLaUs0U9HllX7FVU2DIXVSzNq2HgkiVGYol7BS8u5mdxr9HqssSq7VZ6tOpFnJyZonHzm7E-PO_CfEp3hHIRLo6oLPbCoY5iO2MiqYWljox5g2kSVQo7_PQBCxyO1lMbiHYCW6_s6uMHPSqGiB_td8-ecX9CgjXnTPdd1DR2H4c7hnfKSyoGRGFsZsdDmdcawKXghrxLp9DUK2VgKCR8KVOsNw"
```

Comment:
@asoorm Could you remove this please? The license should not be exposed here. Thanks